### ISSUE TITLE
Super is not defined in ES6 classes

### ISSUE THREAD

**Issue by kompot** [babel/babel-eslint#10]
Given the code

```javascript
import Service from './Service';

export default class AService extends Service {

  constructor(config) {
    super(config);
  }

}
```

output is 
```
6:4  error  'super' is not defined  no-undef
```

**Comment by sebmck** [comment]
Could probably get away with turning this into a `ThisExpression`.

**Comment by sebmck** [comment]
Should be fixed as of 1.0.12, let me know if that did the trick.

**Comment by kompot** [comment]
Thanks, worked like a charm!
